outbox/event-routing-smt/src/main/java/io/debezium/examples/outbox/routingsmt/EventRouter.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.outbox.routingsmt;
+
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.header.Headers;
+import org.apache.kafka.connect.transforms.Transformation;
+
+/**
+ * Takes Kafka Connect schema and JSON payload from a message and expands it into a properly typed record.
+ *
+ * @author Gunnar Morling
+ *
+ * @param <R>
+ */
+public class EventRouter<R extends ConnectRecord<R>> implements Transformation<R> {
+
+    public EventRouter() {
+
+    }
+    @Override
+    public void configure(Map<String, ?> configs) {
+    }
+
+    @Override
+    public R apply(R record) {
+        Struct struct = (Struct) record.value();
```

Comment:
I think here should be a condiiton against tombstone.

Reply:
Done.